oak-run/src/main/java/org/apache/jackrabbit/oak/run/DataStoreCommand.java

```diff
@@ -197,21 +289,31 @@ private static MarkSweepGarbageCollector getCollector(NodeStoreFixture fixture,
                 retriever = new SegmentBlobReferenceRetriever(fileStore);
             }
         }
+        return retriever;
+    }
 
-        ExecutorService service = Executors.newSingleThreadExecutor();
-        closer.register(new ExecutorCloser(service));
+    private static void retrieveBlobIds(GarbageCollectableBlobStore blobStore, File blob)
+            throws Exception {
 
-        String repositoryId = ClusterRepositoryInfo.getId(fixture.getStore());
-        checkNotNull(repositoryId);
+        System.out.println("Starting dump of blob ids");
+        Stopwatch watch = createStarted();
 
-        MarkSweepGarbageCollector collector =
-            new MarkSweepGarbageCollector(retriever, (GarbageCollectableBlobStore) fixture.getBlobStore(), service,
-                dataStoreOpts.getOutDir().getAbsolutePath(), dataStoreOpts.getBatchCount(),
-                SECONDS.toMillis(dataStoreOpts.getBlobGcMaxAgeInSecs()), dataStoreOpts.checkConsistencyAfterGC(),
-                repositoryId, fixture.getWhiteboard(), getService(fixture.getWhiteboard(), StatisticsProvider.class));
-        collector.setTraceOutput(true);
+        Iterator<String> blobIter = blobStore.getAllChunkIds(0);
+        int count = writeStrings(blobIter, blob, false);
 
-        return collector;
+        sort(blob);
+        System.out.println(count + " blob ids found");
+        System.out.println("Finished in " + watch.elapsed(TimeUnit.SECONDS) + " seconds");
+    }
+
+    private static void verboseIds(BlobStoreOptions blobOpts, File readFile, File writeFile) throws IOException {
+        LineIterator idIterator = FileUtils.lineIterator(readFile, Charsets.UTF_8.name());
+
+        try (BurnOnCloseFileIterator<String> iterator =
+                     new BurnOnCloseFileIterator<String>(idIterator, readFile,
+                             (Function<String, String>) input -> VerboseIdLogger.encodeId(input, blobOpts.getBlobStoreType()))) {
+            FileIOUtils.writeStrings(iterator, writeFile, true, log, "Transformed to verbose ids - ");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'FileIOUtils.writeStrings' due to existing static import 'org.apache.jackrabbit.oak.commons.FileIOUtils.writeStrings'](https://app.codacy.com/gh/oak-indexing/jackrabbit-oak/pullRequest?prid=5079914)